common/beerocks/btl/include/btl/btl.h

```diff
@@ -38,6 +38,18 @@ class transport_socket_thread : public socket_thread {
     bool send_cmdu_to_bus(ieee1905_1::CmduMessageTx &cmdu, const std::string &dst_mac,
                           const std::string &src_mac);
 
+    /**
+     * @brief Sends CDMU to transport for dispatching.
+     *
+     * @param cmdu Control Message Data Unit to send.
+     * @param iface_index Network interface index (set to 0 to send on all interfaces).
```

Comment:
Done.
@arnout: Following this same criteria, shall I store interface name instead of interface index in topology database?
#1440 is already merged so before proceeding, I'd create a hotfix PR for the change